README.md

```diff
@@ -1,27 +1,94 @@
+**[jta](#jta-tag-definition)**<a name='jta-tag'> | [ArjunaJTA/maven](ArjunaJTA/maven/), [ArjunaJTA/javax_transaction](ArjunaJTA/javax_transaction/), [ArjunaJTA/jee_transactional_app](ArjunaJTA/jee_transactional_app/), [ArjunaJTA/object_store](ArjunaJTA/object_store/), [ArjunaJTA/recovery](ArjunaJTA/recovery/)
+**[jts](#jts-tag-definition)**<a name='jts-tag'> | [ArjunaJTS/interop](ArjunaJTS/interop/), [ArjunaJTS/jts](ArjunaJTS/jts/), [ArjunaJTS/recovery](ArjunaJTS/recovery/), [ArjunaJTS/standalone](ArjunaJTS/standalone/)
+**[standalone](#standalone-tag-definition)**<a name='standalone-tag'> | [ArjunaCore/txoj](ArjunaCore/txoj/), [ArjunaJTA/maven](ArjunaJTA/maven/), [ArjunaJTA/javax_transaction](ArjunaJTA/javax_transaction/), [ArjunaJTA/object_store](ArjunaJTA/object_store/), [ArjunaJTA/recovery](ArjunaJTA/recovery/), [ArjunaJTS/recovery](ArjunaJTS/recovery/), [ArjunaJTS/standalone](ArjunaJTS/standalone/)
+**[wildfly](#wildfly-tag-definition)**<a name='wildfly-tag'> | [ArjunaJTA/jee_transactional_app](ArjunaJTA/jee_transactional_app/),[ArjunaJTS/interop](ArjunaJTS/interop/), [ArjunaJTS/jts](ArjunaJTS/jts/)
+**[glassfish](#glassfish-tag-definition)**<a name='glassfish-tag'> | [ArjunaJTS/interop](ArjunaJTS/interop/)
+**[iiop](#iiop-tag-definition)**<a name='iiop-tag'> | [ArjunaJTS/standalone](ArjunaJTS/standalone/)
+
+### Tags definition
+
+* **arjuna.core**<a name='arjuna-core-tag-definition'> : demonstrating capabilities of Narayana API,
+  it's helpful for developers want to write a transaction state machine
+  and don't want to start on a green field but rather used battle tested library
+* **narayana.configuration**<a name='narayana-configuration-tag-definition'> : depicting aspects
+  of Narayana configuration and showing options of such configurations
+* **jta**<a name='jta-tag-definition'> : using JTA API to demonstrate transaction processing
+* **jts**<a name='jts-tag-definition'> : using JTS API to demonstrate how the Narayana transaction system
+  could be run and configured to run distributed JTS transactions
+* **standalone**<a name='standalone-tag-definition'> : running as standalone Java SE application
+* **wildfly**<a name='wildfly-tag-definition'> : running as deployment on WildFly application server
+* **glassfish**<a name='glassfish-tag-definition'> : running on GlassFish application server
+* **iiop**<a name='iiop-tag-definition'> : showing how to use ORB API with transaction manager
+
+
 ## Contacting us
 
-We are always happy to talk transactions and how-to use Narayana in exotic and not so exotic environments. If you have ideas for what we can add to the quickstarts to make them more useful please do reach out to us over on our forum:
+We are always happy to talk transactions and how-to use Narayana in exotic and not so exotic environments.
+If you have ideas for what we can add to the quickstarts to make them more useful please do reach out to us over on our forum:
```

Comment:
While we are editing this perhaps replace the link to the forum with a link to http://narayana.io/community/index.html (which includes the forum and the other methods)